lib/openstudio-standards/prototypes/common/objects/Prototype.Model.rb

```diff
@@ -2159,4 +2158,95 @@ def apply_economizers(climate_zone, model)
       end
     end
   end
+
+  # Get building door information to update infiltration
+  #
+  # return [Hash] Door infiltration information
+  def get_building_door_info(model)
+    get_building_door_info = {}
+
+    return get_building_door_info
+  end
+
+  # Metal coiling door code minimum infiltration rate at 75 Pa
+  #
+  # @param [String] Climate zone
+  # @return [Float] Minimum infiltration rate for metal coiling doors
+  def model_door_infil_flow_rate_metal_coiling_cfm_ft2(climate_zone)
+    case climate_zone
+      when 'ASHRAE 169-2006-7A',
+           'ASHRAE 169-2006-7B',
+           'ASHRAE 169-2006-8A',
+           'ASHRAE 169-2006-8B'
+        return 0.4
+      else
+        return 4.4
+    end
+  end
+
+  # Metal rollup door code minimum infiltration rate at 75 Pa
+  #
+  # @param [String] Climate zone
+  # @return [Float] Minimum infiltration rate for metal coiling doors
+  def model_door_infil_flow_rate_rollup_cfm_ft2(climate_zone)
+    return 0.4
+  end
+
+  # Open door infiltration rate at 75 Pa
+  #
+  # @param [String] Climate zone
+  # @return [Float] Open door infiltration rate
+  def model_door_infil_flow_rate_open_cfm_ft2(climate_zone)
+    return 9.7875
+  end
+
+  # Add door infiltration
+  #
+  # @param [OpenStudio:Model::Model] OpenStudio model object
+  # @climate_zone [String] Climate zone
+  # @return [Boolean] Returns true if successful, false otherwise or not applicable
+  def model_add_door_infiltration(model, climate_zone)
```

Comment:
There already exist a `add_door_infiltration` method. The main difference is that the one proposed here aims to handle the infiltration calculations for all prototypes. The current one is different for each prototype and uses hard coded values. While this one is targeting 90.1-2016/9 requirements specifically, it could be used in place the the older one, which could then in turn be depreciated.

Reply:
This triggered the regression test to fail. I don't think we need to update the test results since we know exactly what the discrepancy is.